pimcore/lib/Pimcore/Controller/Action/Frontend.php

```diff
@@ -345,7 +345,11 @@ public function initTranslation() {
         if(!$translate) {
             // setup \Zend_Translate
             try {
-                $locale = \Zend_Registry::get("Zend_Locale");
+                if(\Zend_Registry::isRegistered("Zend_Locale")) {

```

Comment:
I don't understand why you automatically use the first language? 
\Zend_Registry::get("Zend_Locale"); is wrapped by try/catch so nothin happens when no locale is set, you only get a message in the log. I don't think it's a good idea to automatically set the first language. 

Please remove this change from the pull request, then I can merge it.
Thank  you very much!